packages/react-components/components/ContactCircle.tsx

```diff
@@ -25,7 +27,7 @@ const getNameInitial = (name: string) => name.charAt(0).toLocaleUpperCase()
 
 export default class ContactCircle extends React.PureComponent<Props> {
   getContactCircleInner = () => {
-    const { contact, name, size, preferNameInitial, thumbnailPath } = this.props
+    const { contact, name, size, preferNameInitial, thumbnailPath, recipientKind } = this.props
```

Comment:
Can we try to make this work without relying on RecipientKind? Recipients are a concept that we haven't really shared outside the mobile package yet. 
If we are going to have it here, I feel we should go all the way with it, meaning we should replace this ContactCircle with a RecipientCircle component and rewrite some of its internals accordingly.